### PR TITLE
Fixes #9717: list-compatible-inputs.ps1 doesn't support capabilities

### PR DESCRIPTION
**Pull request by ncharles**
https://www.rudder-project.org/redmine/issues/9717

### CODE REVIEW

File: tree/10_ncf_internals/list-compatible-inputs.ps1 [hunk 1]
```diff
@@ -5,24 +5,84 @@ param(
...
 $path="$framework_path\$nn_directory"
 
-# Create an empty exclude list
+# We consider that the Windows agent doesn't have any capability
+
+$version_regex="([0-9]+).([0-9]+)*"
+$agent_version_regex="^#[\s]*@agent_version[\s]*(>=|<)[\s]*$($version_regex)"
+# capture capabilities, and remainder of line
+$agent_requirements_regex="^#[\s]*@agent_requirements[\s]*""capabilities""[\s]*:[\s]*\[[\s]*(.*)[\s]*\](.*)"
+$reminder_regex="[\s]*(&&|\|\|)[\s]*""agent_version""[\s]*(>=|<)[\s]*$($version_regex)"
```
Comment (peckpeck) [inline tree/10_ncf_internals/list-compatible-inputs.ps1]: "reminder" ? 
What does it mean ?

Comment (ncharles) [inline tree/10_ncf_internals/list-compatible-inputs.ps1]: it is what remains on the line, after capabilities. Name is not correct :/

**Comment by ncharles** [comment]
Commit modified

File: tree/10_ncf_internals/list-compatible-inputs.ps1 (hunk 1 above)
Comment (peckpeck) [inline tree/10_ncf_internals/list-compatible-inputs.ps1]: Remainder is what remains.
Reminder is what you put put on your fridge to remind you something.

**Pull request merged**